exist-core/src/main/java/org/exist/util/serializer/HTML5Writer.java

```diff
@@ -38,13 +38,13 @@ public class HTML5Writer extends XHTML5Writer {
 
     /**
      * Holds the names of the attributes that are considered boolean
-     * according to http://www.w3.org/TR/html51/single-page.html.
-     * 
+     * according to {@link <a href="http://www.w3.org/TR/html51/single-page.html">...</a>}
+     *
      * The value of these attributes are written if they equal the
      * name of the attribute. For example: checked="checked" will be
      * written as checked.
-     * 
-     * See https://github.com/eXist-db/exist/issues/777 for details. 
+     *
+     * See <a href="https://github.com/eXist-db/exist/issues/777">...</a> for details.
```

Comment:
see above
Additionally, shouldn't it be either `{@link ... }` or `<a href="...` ?

Reply:
You're correct, it should. Fix on the way.